Dockerfile

```diff
@@ -10,7 +10,7 @@ WORKDIR /usr/src/app
 
 # Install packages depending on ENV arg from docker-compose
 ARG BASE_PACKAGES="gcc libpq-dev libjpeg62-turbo-dev libsqlite3-0 zlib1g-dev"
-ARG DEV_PACKAGES="firefox-esr"
+ARG DEV_PACKAGES=""
```

Comment:
If we're dropping that package, I think we can remove all of the differentiation between packages to install per environment in the Dockerfile!

Reply:
Removed extra Dockerfile logic!